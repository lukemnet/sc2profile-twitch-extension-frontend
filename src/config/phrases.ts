export default {
  FORM_VALIDATION_URL_TOO_SHORT: 'Please provide correct StarCraft II profile URL.',
  FORM_VALIDATION_URL_INVALID: 'Profile URL is invalid.',
  FORM_VALIDATION_DUPLICATE_URLS: 'Duplicate profiles detected. Please review your configuration before saving.',
};
