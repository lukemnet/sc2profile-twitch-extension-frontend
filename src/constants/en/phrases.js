module.exports = {
  en: {
    config: {
      loading: 'Loading extension data...',
      fields: {
        server: {
          label: 'Server',
          choose: 'Choose a server',
          eu: 'EU - Europe',
          us: 'US - Americas & Oceania',
          kr: 'KR - Korea',
        },
        playerid: {
          label: 'Player ID',
        },
        region: {
          label: 'Region',
        },
        name: {
          label: 'Name',
        },
        profileUrl: {
          label: 'Battle.net profile URL',
        },
        submit: {
          label: 'Save',
        },
      },
      validation: {
        fieldCannotBeEmpty: 'Field cannot be empty',
        selectServer: 'Select server',
      },
      manual: {
        button: {
          label: 'Instructions - click to view â†“',
        },
        content: {},
      },
      messages: {
        config_get_no_config_found_warning: 'No StarCraft II profile found. Fill the form below to make the extension work.',
        config_get_error: 'Ooops, an error occured while getting config data. Please refresh the page and try again.',
        config_get_inprogress: 'Loading your configuration. Please wait...',
        config_get_success: 'Extension configured successfully! Want to showcase a different StarCraft II profile? Fill the form below.',
        config_save_success: 'Your configuration has been saved successfully! Go to your channel page and add the extension to your panels.',
        config_save_inprogress: 'Saving your configuration. Please wait...',
        config_save_error: 'Erro while saving configuration. Please refresh the page and try again.',
      },
    },
  },
};
