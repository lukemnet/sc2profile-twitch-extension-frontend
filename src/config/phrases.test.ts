import phrases from './phrases';

it('exports phrases config object', () => {
  expect(phrases).toMatchSnapshot();
});
