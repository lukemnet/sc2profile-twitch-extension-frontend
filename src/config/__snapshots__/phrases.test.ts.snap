// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports phrases config object 1`] = `
Object {
  "FORM_VALIDATION_DUPLICATE_URLS": "Duplicate profiles detected. Please review your configuration before saving.",
  "FORM_VALIDATION_URL_INVALID": "Profile URL is invalid.",
  "FORM_VALIDATION_URL_TOO_SHORT": "Please provide correct StarCraft II profile URL.",
}
`;
